### PR TITLE
ditribute crc dpos reward to corresponding cr members

### PR DESCRIPTION
**Pull request by raindust**

### CODE REVIEW

File: dpos/state/originarbiter.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2017-2019 The Elastos Foundation
+// Use of this source code is governed by an MIT
+// license that can be found in the LICENSE file.
+//
+
+package state
+
+import (
+	"github.com/elastos/Elastos.ELA/core/contract"
```
Comment (RainFallsSilent) [inline dpos/state/originarbiter.go]: format

File: dpos/state/originarbiter_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017-2019 The Elastos Foundation
+// Use of this source code is governed by an MIT
+// license that can be found in the LICENSE file.
+//
+
+package state
+
+import (
+	"bytes"
+	"github.com/elastos/Elastos.ELA/crypto"
```
Comment (RainFallsSilent) [inline dpos/state/originarbiter_test.go]: format

File: dpos/store/eventstore_test.go [hunk 3]
```diff
@@ -6,7 +6,7 @@
 package store
 
 import (
-	"crypto/rand"
+	"github.com/elastos/Elastos.ELA/crypto"
```
Comment (RainFallsSilent) [inline dpos/store/eventstore_test.go]: format

File: blockchain/txvalidator_specailtx_test.go [hunk 4]
```diff
@@ -7,6 +7,7 @@ package blockchain
 
 import (
 	"bytes"
+	"github.com/elastos/Elastos.ELA/common/log"
```
[the remark this answers is not in the document]
Comment (raindust) [reply blockchain/txvalidator_specailtx_test.go]: fixed

File: dpos/state/arbitermember_test.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017-2019 The Elastos Foundation
+// Use of this source code is governed by an MIT
+// license that can be found in the LICENSE file.
+//
+
+package state
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
```
[the remark this answers is not in the document]
Comment (raindust) [reply dpos/state/arbitermember_test.go]: fixed

File: dpos/state/originarbiter_test.go (hunk 2 above)
Comment (raindust) [reply dpos/state/originarbiter_test.go]: fixed

File: dpos/store/eventstore_test.go (hunk 3 above)
Comment (raindust) [reply dpos/store/eventstore_test.go]: fixed

File: dpos/state/originarbiter.go (hunk 1 above)
Comment (raindust) [reply dpos/state/originarbiter.go]: fixed

File: dpos/state/arbitermember.go [hunk 6]
```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2017-2019 The Elastos Foundation
+// Use of this source code is governed by an MIT
+// license that can be found in the LICENSE file.
+//
+
+package state
+
+import (
+	"errors"
+	"github.com/elastos/Elastos.ELA/common"
```
[the remark this answers is not in the document]
Comment (raindust) [reply dpos/state/arbitermember.go]: fixed

File: dpos/state/arbitrators_test.go [hunk 7]
```diff
@@ -7,7 +7,7 @@ package state
 
 import (
 	"bytes"
-	"crypto/rand"
+	"github.com/elastos/Elastos.ELA/crypto"
```
Comment (yz89) [inline dpos/state/arbitrators_test.go]: format

Comment (raindust) [reply dpos/state/arbitrators_test.go]: fixed

**Comment by raindust** [comment]
resolve conflicts after merge from release_v0.4.0

**Pull request merged**